Codeinwp/optimole-wp
# Add option to disable strip_metadata

### All Submissions:

 
### Changes proposed in this Pull Request:
This adds a toggle in Optimole Dashboard to disable strip_metadata. This PR requires for this to be merged: https://github.com/Codeinwp/optimole-wp/pull/534
 
Closes https://github.com/Codeinwp/optimole-service/issues/695

### How to test the changes in this Pull Request:

1. First you need to upload an image that has metadata to your WP instance. A good idea is just to use your phone camera to take a picture and upload it to WP.
2. Check for metadata once it's been processed through Optimole and by default it shouldn't exist.
3. Now disable the metadata option from Optimole Dashboard and repeat the second step and this time it should exist.

To check metadata you can use a website like this: https://jimpl.com/ - or just download it to your laptop and check the image for info/properties.

**Also, you need to test it on using optimole-test plugin.**

### Other information:
